console/src/assets/i18n/en.json

```diff
@@ -2514,13 +2515,104 @@
     "URLS": "URLs",
     "CONFIGURATION": "Configuration",
     "TOKEN": "Token Settings",
+    "QUICKSTART": "Quickstart",
```

Comment:
Do we fall back to english if a translation is not avaliable in other languages?